docs/content/manual/pre-release/ha/partial-engine-deployment.md

```diff
@@ -22,7 +22,7 @@ https://github.com/longhorn/longhorn/issues/2081
 1. Detach `vol-1`
 1. Verify that Longhorn cannot attach `vol-1` to `node-1` since there is no engine image on `node-1`
 1. Check that all functions (attach to other nodes, detach, snapshot, backup, expand, restore, creating DR volume, ... ) are working ok for `vol-1`
-1. Verify that `vol-2` can be attached to any nodes except for `node-1`
+1. Verify that `vol-2` cannot be attached to any nodes because one of its replica is sitting on the `node-1` which doesn't have the engine image
```

Comment:
IIRC, after the fix of longhorn/longhorn#4197, the volume can be attached even if there are failed replicas (caused by lacking the engine image for example) during attachment. 
Will the engine image get stuck in state `deploying` and block the attachment?

Reply:
After command `kubectl get pods -n longhorn-system`, from the response it can not find stuck pods ( all in running status)

Reply:
I think @shuo-wu was asking about the engine image CR in deploying state. I think it should be in deploying state because there is no engine image pod on the trained node, but the longhorn manager pod is still there? @cchien816

Reply:
Yes by `kubectl get engineimages.longhorn.io` can see the engine image CR is in deploying status
```
NAME          STATE       IMAGE                                   REFCOUNT   BUILDDATE   AGE
ei-1b463ad1   deploying   longhornio/longhorn-engine:v1.3.1-rc2   5          24h         41m
```

Reply:
Then will this `deployment` engine image block the volume attachment?